homeassistant/components/jvc_projector/binary_sensor.py

```diff
@@ -0,0 +1,47 @@
+"""Binary Sensor platform for JVC Projector integration."""
+
+from __future__ import annotations
+
+from jvcprojector import const
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import JvcProjectorDataUpdateCoordinator
+from .const import DOMAIN
+from .entity import JvcProjectorEntity
+
+ON_STATUS = [const.ON, const.WARMING]
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the JVC Projector platform from a config entry."""
+    coordinator: JvcProjectorDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_entities([JvcBinarySensor(coordinator)])
+
+
+class JvcBinarySensor(JvcProjectorEntity, BinarySensorEntity):
+    """The entity class for JVC Projector integration."""
+
+    _attr_name = "Power"
+
+    icon = "mdi:power"
```

Comment:
Please set a translation key and add the entity name to strings.json and add the icon to icons.json